### PR TITLE
chore(vision): Add some missing samples from the tmp-generated-samples branch

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

File: google-cloud-vision/samples/async_batch_annotate_images.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 Google LLC
```
Comment (quartzmo) [inline google-cloud-vision/samples/async_batch_annotate_images.rb:1]: 2020? (for all three files)

File: google-cloud-vision/samples/async_batch_annotate_images.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# sample-metadata
+#   title: Async Batch Image Annotation
+#   description: Perform async batch image annotation
+
+require "google/cloud/vision"
```
Comment (quartzmo) [inline google-cloud-vision/samples/async_batch_annotate_images.rb:19]: Should these requires be inside the region tags (usually also inside the sample method?)

File: google-cloud-vision/samples/async_batch_annotate_images.rb (hunk 1 above)
Comment (dazuma) [reply google-cloud-vision/samples/async_batch_annotate_images.rb]: yeah fixed

File: google-cloud-vision/samples/async_batch_annotate_images.rb (hunk 2 above)
Comment (dazuma) [reply google-cloud-vision/samples/async_batch_annotate_images.rb]: Oh. Yeah good catch, I think so. I copied these verbatim from the original orphaned branch on ruby-docs-samples and didn't pay enough attention. Fixed.

**Pull request merged**